### DISCUSSION TITLE
Storybook + twin macro

### DISCUSSION THREAD

**Discussion by joeynimu** [ben-rogerson/twin.macro#658] [Q&A]
Hey Folks,

I am having an issue when trying to use twin with storybook where when running `start-storybook -p 6006` components that are using extended classes in `tailwind.config.js` are erroring and it seems like it picks up the default classes in tailwind for the error suggests those classes. 

```
✕ text-yellow-400 was not found
Did you mean text-yellow?
```

I am guessing there's a missing configuration? I also found [this stack-overflow](https://stackoverflow.com/questions/71685795/issues-with-twin-macro-in-a-monorepo-where-storybook-is-at-the-root) issue that's somehow similar to mine just for more context

**Comment by ben-rogerson** [comment]
Hey there
Twin should only suggest classes it's found when it merges your config with the defaults.
Could you let us know the content of your tailwind.config?
